django/db/backends/sqlite3/base.py

```diff
@@ -27,13 +30,30 @@
 from .schema import DatabaseSchemaEditor                    # isort:skip
 
 
+if settings.DEBUG:
```

Comment:
`sqlite3.enable_callback_tracebacks(settings.DEBUG)`?

Reply:
That would be cleaner.